mongorestore/mongorestore_test.go

```diff
@@ -1970,7 +1970,13 @@ func TestRestoreTimeseriesCollections(t *testing.T) {
 				numIndexes++
 			}
 
-			So(numIndexes, ShouldEqual, 0)
+			if (restore.serverVersion.GTE(db.Version{6, 3, 0})) {
+				Convey("--noIndexRestore should make no impact on the default meta, time index for time-series collections if server version >= 6.3.0", func() {
```

Comment:
Can we change the message to be more along the lines of `... should build the index on meta, time by default for time-series collections...`

Reply:
done